packages/marketplace/src/components/Modals/Settings/ConfigRow.tsx

```diff
@@ -2,15 +2,18 @@ import React from "react";
 import { Config } from "../../../types/marketplace-types";
 
 import Toggle from "../../Toggle";
-
+import SortBox from "../../Sortbox";
 const ConfigRow = (props: {
   name: string;
   storageKey: string;
   modalConfig: Config;
   clickable?: boolean;
   updateConfig: (CONFIG: Config) => void;
+  type: string | null;
+  options: string[] | null;
```

Comment:
Pass parameters as optional so you don't have to do all that funky thing below lol
```suggestion
  type?: string;
  options?: string[];
```